messagebus/messagebus.go

```diff
@@ -257,6 +265,26 @@ func (mb *MessageBus) doDeliver(ctx context.Context, msg core.Parcel) (core.Repl
 	defer readBarrier(ctx, &mb.globalLock)
 	ctx, _ = inslogger.WithField(ctx, "msg_type", msg.Type().String())
 	inslogger.FromContext(ctx).Debug("MessageBus.doDeliver starts ...")
+	_, ok := conveyorReadyTypes[msg.Type()]
+	if ok {
+		f := mb.futureManager.Create()
+		event := ConveyorPendingMessage{Msg: msg, F: f}
+		err := mb.Conveyor.SinkPush(msg.Pulse(), event)
+		if err != nil {
+			f.Cancel()
+			err := errors.Wrapf(err, "error while calling Conveyor.SinkPush")
+			inslogger.FromContext(ctx).Error(err)
+			return nil, err
```

Comment:
wrap

Reply:
I did(3 lines before, wrap and then log&return)